src/main/scala/algolia/inputs/InsightsEvent.scala

```diff
@@ -0,0 +1,76 @@
+package algolia.inputs
+
+import algolia.AlgoliaUtils
+import org.json4s.CustomSerializer
+
+case class InsightsEvent(
+    eventType: String,
+    eventName: String,
+    index: String,
+    userToken: String,
+    timestamp: Long = AlgoliaUtils.now(),
```

Comment:
Can't we set it as `Option[Long] = None` ?

Reply:
Right thanks 👍 Completely forgot it in Scala

Reply:
Actually, it will simply the tests 🎉